packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -77,17 +79,44 @@ async function ensureLockedRecord(
   return attestationRecord
 }
 
+async function purgeExpiredRecords() {
+  const attestations = await kit.contracts.getAttestations()
+  const expiryTimeInSeconds = (await attestations.attestationExpiryBlocks()) * 5
```

Comment:
Let's cache this value. It would only change through governance, so almost never.